corecito_account.py

```diff
@@ -33,6 +36,7 @@ def __init__(self, config=None):
       binance = Binance(public_key = self.api_key, secret_key = self.api_secret, sync=True)
       self.account = binance.b
       self.pair = config['binance_trading_pair']
+      self.pair_name = self.pair.replace('_', '/')
```

Comment:
Maybe for CRO the pair is BTC_ETC (with an underscore), but for Binance the pairs don't have an underscore (correct me if I'm wrong).
For example, I'm using the pair BTCEUR, so the method `replace` here will actually do nothing 😅

Reply:
Correct, so in the case of Binance it'll do no harm.